### PR TITLE
Multicluster - Health for applications/services/workloads

### LINKED ISSUE #6026: Multicluster - Health calculation for workload/app/svc
Should we include the cluster parameter for the calculation of the health service/app? 

The desired behavior is filter the cluster so we get the metrics for a specific name for a cluster and a namespace, but this is breaking the 
health calculations, at least for cypress tests.

Ref. MC Services List: https://github.com/kiali/kiali/pull/5998#pullrequestreview-1381147997
Ref. https://github.com/kiali/kiali/pull/5990

### PR DESCRIPTION
**Pull request by leandroberetta**
Resolves #6026 

Epic: #5618

### CODE REVIEW

File: prometheus/client.go [hunk 1]
```diff
@@ -169,19 +169,19 @@ func (in *Client) GetNamespaceServicesRequestRates(namespace, cluster string, ra
 // be inflated due to duplication, and therefore should be used mainly for calculating ratios
 // (e.g total rates / error rates).
 // Returns (in, error)
-func (in *Client) GetServiceRequestRates(namespace, service, ratesInterval string, queryTime time.Time) (model.Vector, error) {
+func (in *Client) GetServiceRequestRates(namespace, cluster, service, ratesInterval string, queryTime time.Time) (model.Vector, error) {
 	log.Tracef("GetServiceRequestRates [namespace: %s] [service: %s] [ratesInterval: %s] [queryTime: %s]", namespace, service, ratesInterval, queryTime.String())
 	if promCache != nil {
-		if isCached, result := promCache.GetServiceRequestRates(namespace, service, ratesInterval, queryTime); isCached {
+		if isCached, result := promCache.GetServiceRequestRates(namespace, kubernetes.KialiTokenForHomeCluster, service, ratesInterval, queryTime); isCached {
```
Comment (hhovsepy) [inline prometheus/client.go:175]: Shouldn't it be `cluster` instead of `kubernetes.KialiTokenForHomeCluster`?

Comment (hhovsepy) [reply prometheus/client.go:175]: Because for App and Workloads it is passed `cluster` param.

**Review by josunect — COMMENTED** [review]
Thanks, @leandroberetta !

I've added a couple of comments

**Review by josunect — APPROVED** [review]
LGTM, thanks @leandroberetta !

**Pull request merged**